woocommerce-gateway-ebanx/controllers/class-wc-ebanx-api-controller.php

```diff
@@ -29,6 +29,50 @@ public function dashboard_check() {
 	}
 
 	/**
+	 * Cancels an open cash payment order with "On hold" status
+	 *
+	 * @return void
+	 */
+	public function cancel_order($order_id, $user_id) {
+		$user_id = intval($user_id);
+		$order_id = intval($order_id);
+		$order = new WC_Order( $order_id );
+		if ($user_id !== get_current_user_id()
+		    || $order->get_status() !== 'on-hold'
+		    || !in_array($order->get_payment_method(), WC_EBANX_Constants::$CASH_PAYMENTS)
+			) {
+				wp_redirect( get_site_url() );
+				return;
+		}
+
+		$data = array(
+			'hash' => get_post_meta($order_id, '_ebanx_payment_hash', true)
+		);
+
+		\Ebanx\Config::set(array(
+			'integrationKey' => $this->get_integration_key(),
+			'testMode'       => $this->is_sandbox(),
+		));
+
+		try {
+			$request = \Ebanx\EBANX::doCancel($data);
+
+			if ($request->status === 'SUCCESS') {
+				$order->update_status('cancelled', 'Cancelled by customer');
```

Comment:
Change it to `__('EBANX: Cancelled by customer', 'woocommerce-gateway-ebanx')` and translate it